src/commands/githubActions/bumpUpstream/index.ts

```diff
@@ -175,6 +175,55 @@ Compose - ${JSON.stringify(compose, null, 2)}
     body: getPrBody(versionsToUpdate)
   });
 
+  // get the list of branches of the repo
+  const allBranches = await thisRepo.listBranches();
+
+  // to obtain the url and other info of the new PR
+  const newPr = await thisRepo.getOpenPrsFromBranch({ branch });
+  const newPrUrl = newPr[0].html_url; // get the link of the new PR
```

Comment:
Check if `newPr[0]` exists first

Reply:
Now it checks if exists, in that case throw an error